mzLib/mzLib.nuspec

```diff
@@ -22,8 +22,8 @@
         <dependency id="OpenMcdf.Extensions" version="2.2.1.3" />
         <dependency id="SharpLearning.Optimization" version="0.28.0" />
       </group>
-      <group targetFramework="net6.0-windows7.0">
-        <!--net6.0-windows7.0 target with OxyPlot-->
+      <group targetFramework="net8.0-windows8.0">
```

Comment:
I think this can be `net8.0-windows`

Reply:
this change should be made to net8.0-windows7.0